components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt.ui/src/main/java/org/wso2/carbon/identity/functions/library/mgt/ui/client/FunctionLibraryManagementServiceClient.java

```diff
@@ -0,0 +1,149 @@
+package org.wso2.carbon.identity.functions.library.mgt.ui.client;
+
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.client.Options;
+import org.apache.axis2.client.ServiceClient;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.axis2.transport.http.HTTPConstants;
+import org.wso2.carbon.identity.functions.library.mgt.model.xsd.FunctionLibrary;
+import org.wso2.carbon.identity.functions.library.mgt.stub.FunctionLibraryManagementAdminServiceFunctionLibraryManagementException;
+import org.wso2.carbon.identity.functions.library.mgt.stub.FunctionLibraryManagementAdminServiceStub;
+
+import java.rmi.RemoteException;
+
+/**
+ * Function library management service client.
+ */
+public class FunctionLibraryManagementServiceClient {
+    FunctionLibraryManagementAdminServiceStub stub;
+
+    /**
+     *
```

Comment:
Description of the method and params. 

Add for others as well

Reply:
added